src/main/java/team/creative/creativecore/common/config/group/Usergroup.java

```diff
@@ -17,8 +18,7 @@ public Usergroup() {
     }
     
     public Usergroup(PlayerSelector... selectors) {
-        for (PlayerSelector selector : selectors)
-            filters.add(selector);
+        filters.addAll(Arrays.asList(selectors));
```

Comment:
i do the change by performance reasons, i check again the check (after sleep 12 hours) and i see performance is worse. mosly because it copies the array 3 times. for some reason i ignore the fact it is on constructor, so i change the add all to ``Array.asList()`` which literally creates a new ArrayList instance using the same argumented array.

resulting code looks like this and is much better than before
![image](https://github.com/CreativeMD/CreativeCore/assets/29090346/8449121e-5217-464d-bb89-43bda530ae02)